ubcpi/static/css/ubcpi.css

```diff
@@ -0,0 +1,495 @@
+  float: left;
+  clear: both;
+  margin-bottom: 5px;
+  display: block;
+}
+
+    .ubcpi-label img {
+        max-width: 100%;
+        height: auto;
+    }
+
+.ubcpi-label.ubcpi-answer {
+    float: none;
+}
+
+.results-container .ubcpi-label {
+    margin-bottom: 15px;
+}
+
+.ubcpi-field{
+  display: block;
+}
+
+.ubcpi-field[type="radio"]{
+  float: left;
+  margin: 4px;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+textarea.ubcpi-field{
+  width: 100%;
+  height: 100px;
+  margin: 1em 0;
+  background: #fff;
+  border: 1px solid #c8c8c8;
+  border-radius: 3px;
+  box-shadow: 0 1px 0 0 rgba(255,255,255,0.6),inset 0 0 3px 0 rgba(0,0,0,0.1);
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  font: normal 1em "Open Sans",Verdana,Geneva,sans-serif,sans-serif;
+  padding: 5px 12px;
+  vertical-align: top;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*.choicegroup .ubcpi_submit{
+  border: 1px solid #cacaca;
+  border-radius: 3px;
+  box-shadow: inset 0 1px 0 0 #fff;
+  color: #333;
+  display: inline-block;
+  font-size: inherit;
+  font-weight: bold;
+  background-color: #eee;
+  background-image: -webkit-linear-gradient(#eee,#d2d2d2);
+  background-image: linear-gradient(#eee,#d2d2d2);
+  padding: 7px 18px;
+  text-decoration: none;
+  text-shadow: 0 1px 0 #f8f8f8;
+  background-clip: padding-box;
+  border: 1px solid #cfc6c6;
+  border-radius: 3px;
+  box-shadow: inset 0 1px 0 0 #fff;
+  color: #333;
+  display: inline-block;
+  font-size: inherit;
+  font-weight: bold;
+  background-color: #eee;
+  background-image: -webkit-linear-gradient(#eee,#d6cece);
+  background-image: linear-gradient(#eee,#d6cece);
+  padding: 7px 18px;
+  text-decoration: none;
+  text-shadow: 0 1px 0 #f9f8f8;
+  background-clip: padding-box;
+  font-size: 0.8125em;
+}
+
+.choicegroup input[type="button"]:disabled{
+  opacity: 0.5;
+  cursor: not-allowed;
+}*/
+
+.choicegroup .ubcpi_submit {
+  padding: 10px 40px;
+  background: #33a6dc;
+  border: 1px solid #2b8dbb;
+  color: #fff;
+  border-radius: 4px;
+  display: inline-block;
+  background-image: none !important;
+  box-shadow: none !important;
+  cursor: pointer;
+  text-decoration: none;
+  transition: color 0.25s ease-in-out,background 0.25s ease-in-out,box-shadow 0.25s ease-in-out;
+  -webkit-transition: color 0.25s ease-in-out,background 0.25s ease-in-out,box-shadow 0.25s ease-in-out;
+  -moz-transition: color 0.25s ease-in-out,background 0.25s ease-in-out,box-shadow 0.25s ease-in-out;
+  box-sizing: border-box;
+  font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+  font-weight: 600;
+  text-shadow: none;
+}
+
+.choicegroup .ubcpi_submit:hover,
+.choicegroup .ubcpi_submit:active,
+.choicegroup .ubcpi_submit:focus {
+  background: #00a7f6;
+  background-color: #00a7f6 !important;
+  border-color: #00a7f6;
+  background-image: none !important; /* Needed because edX is very specific here, sadly */
```

Comment:
Is there a rule in edx-platform we could perhaps change to avoide the `!important`s in this file?